### PR TITLE
Combind data

### PR DESCRIPTION
**Pull request by MSBradshaw**
Added the test data sets into the repo
Added several new learner functions
Created a dataframe in main.py that has the combined protein and clinical data and the mismatch labels as training labels

### CODE REVIEW

File: learner_functions.py [hunk 1]
```diff
@@ -48,3 +49,26 @@ def train_sgd(data, labels):
 
     scores = cross_val_score(sgd, data, labels, cv=cv, scoring=SCORING_METHOD)
     print(scores)
+
+def train_nc(data,labels):
+    nc = NearestCentroid(metric='manhattan')
```
Comment (jdayton3) [inline learner_functions.py]: If we want, we can use the `train_classifier()` function to wrap these in.

File: main.py [hunk 2]
```diff
@@ -3,14 +3,34 @@
...
+protein.columns = protein.iloc[0]
+protein = protein.iloc[1:]
 
 #impute missing values with 0
 protein = protein.fillna(0)
 clinical = clinical.fillna(0)
 
-#remove the sample labels and transform the data such that proteins are features
-protein_data = protein.iloc[:,1:].T
+#combind teh clinical and the protein data
```
Comment (jdayton3) [inline main.py]: Here and in other places in the file, I think "combind" should be "combined"?

And "teh" -> "the" 🙂

**Review by jdayton3 — CHANGES_REQUESTED** [review]

**Comment by MSBradshaw** [comment]
I made the requested changes

**Review by jdayton3 — APPROVED** [review]

**Comment by jdayton3** [comment]
@MSBradshaw Good to merge!

**Pull request merged**